<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(178, 68, 173);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>341</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
background-style: solid;
border-width: 1px;
border-radius: 10px;
border-color: rgb(125, 125, 125);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Consolas&quot;;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 2px solid rgb(37, 39, 48);
	border-radius: 20px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	padding-left: 20px;
	padding-right: 20px;

}
QLineEdithover {
	border: 2px solid rgb(46, 50, 62);
}
QLineEditFocus {
	border: 2px solid rgb(85, 170, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>310</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Consolas&quot;;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_field">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 2px solid rgb(37, 39, 48);
	border-radius: 20px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	padding-left: 20px;
	padding-right: 20px;

}
QLineEdithover {
	border: 2px solid rgb(46, 50, 62);
}
QLineEditFocus {
	border: 2px solid rgb(85, 170, 255);
}</string>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="password_check">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: italic 10pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>show</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Arial&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>-60</x>
      <y>0</y>
      <width>991</width>
      <height>1001</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../wallpapers/0006.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>420</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 2px solid rgb(37, 39, 48);
border-radius: 20px;
padding-left: 20px;
padding-right: 20px;

QLineEdithover {
	border: 2px solid rgb(46, 50, 62);
}
QLineEditFocus {
	border: 2px solid rgb(85, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <zorder>label_4</zorder>
   <zorder>graphicsView</zorder>
   <zorder>label_2</zorder>
   <zorder>username</zorder>
   <zorder>label_3</zorder>
   <zorder>password_field</zorder>
   <zorder>password_check</zorder>
   <zorder>label</zorder>
   <zorder>login_button</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
